contrib/system-setup.bat

```diff
@@ -0,0 +1,75 @@
+@echo off
+SETLOCAL ENABLEEXTENSIONS
+
+set output="c:\system-setup.log"

```

Comment:
I suppose this is why it has to be run as Administrator on "modern" windows? Maybe the file could be written in user directory or user temporary directory (`%TEMP%` ?) and then automatically opened with notepad?